guides/common/modules/proc_preparing-for-disaster-recovery-with-two-active-project-servers.adoc

```diff
@@ -0,0 +1,51 @@
+Configure both servers to operate independently in their respective data centers, but ensure that their content does not drift apart over time.
+
+.Procedure
+. Back up your {ProjectServer}.
+For more information, see xref:backing-up-{project-context}-server-and-{smart-proxy-context}_admin[].
+. Restore the backup on a system that will serve as your other {ProjectServer}.
+For more information, see xref:restoring-{project-context}-server-or-{smart-proxy-context}-from-a-backup_admin[].
++
+[NOTE]
+====
+Each server must have a distinct hostname and IP address.
+This enables you to re-register hosts if one of the servers fails.
+====
+. Ensure that content on your servers remains consistent:
+* If you want both servers to manage content synchronization and content view creation, follow these guidelines to prevent content drift:
+** Regularly synchronize repositories on both servers.
+You can use the following Ansible modules to automate repository synchronization: `{ansible-namespace}.repository_sync` and `{ansible-namespace}.sync_plan`.
+** Ensure that content views on both servers match.
+* If you want one server to manage content synchronization and content view creation, use one of these features to prevent content drift:
+** If your disaster recovery site has network access to your primary site, use {ISS} (ISS) to ensure your disaster recovery server synchronizes its content from the primary server.
+** If your disaster recovery site does not have network access to your primary site, synchronize content by using export and import.
+* If you want one server to manage only content view creation but not content synchronization, you can configure the other server or multiple other servers to import content views from the first server but synchronize content from repositories.
+//      |------------ Foreman/Katello for CVs only ---------------|
+//                /                               \
+//              /                                   \
+//            /                                       \
+// |--Foreman/Katello for Hosts --|     |--Foreman/Katello for hosts--|
+. Register hosts to your servers so that each server manages hosts in its respective data center.
+For example, register all hosts in _My_Data_Center_1_ to one {ProjectServer} and all hosts in _My_Data_Center_2_ to the other {ProjectServer}.
+. Automate running the `{foreman-maintain} health check` command on both servers.
+The health check verifies whether the servers remain fully operational.
+
+.Verification
+Perform this test in an isolated staging environment:
+
+. Mimic a full outage on one of your servers.
+To verify that the server is inaccessible, you can turn the machine off, halt the virtual machine (VM) if your server runs on a VM, or isolate the machine by using a firewall.
+. Verify that your `{foreman-maintain} health check` automation reported an error.
+. Re-register all hosts from the inaccessible server to the other server.
```

Comment:
```suggestion
. Register the hosts of the inaccessible server to the other server.
```
I recommend this wording because "re-register ... from" sounds strange.

Reply:
As explained above, I'm keeping "re-register". No change.